e2e/helm.e2e.spec.ts

```diff
@@ -0,0 +1,97 @@
+import path from 'path';
+import {
+  ElectronApplication, BrowserContext, _electron, Page, Locator
+} from 'playwright';
+import { test, expect } from '@playwright/test';
+import {
+  createDefaultSettings, kubectl, helm, tearDownHelm, playwrightReportAssets
+} from './utils/TestUtils';
+
+let page: Page;
+
+test.describe.serial('Helm Deployment Test', () => {
+  let electronApp: ElectronApplication;
+  let context: BrowserContext;
+
+  const mainTitleSelector = '[data-test="mainTitle"]';
+
+  test.beforeAll(async() => {
+    createDefaultSettings();
+
+    electronApp = await _electron.launch({
+      args: [
+        path.join(__dirname, '../'),
+        '--disable-gpu',
+        '--whitelisted-ips=',
+        '--disable-dev-shm-usage',
+      ]
+    });
+    context = electronApp.context();
+
+    await context.tracing.start({ screenshots: true, snapshots: true });
+    page = await electronApp.firstWindow();
+  });
+
+  /**
+   * helm teardown
+   * It should run outside of the electronApp.close(), just to make sure the teardown won't
+   * affect the shutdown process in case of exceptions/errors.
+   */
+  test.afterAll(async() => {
+    await tearDownHelm();
+  });
```

Comment:
```suggestion
  test.afterAll(tearDownHelm);
```
At this point you're creating a function, and all that does is call a different function… no need to have the indirection.

Reply:
Thanks for the heads up.